lib/app/features/feed/stories/views/components/story_viewer/components/core/story_viewer_action_buttons.dart

```diff
@@ -30,11 +31,20 @@ class StoryViewerActionButtons extends ConsumerWidget {
           _SoundButton(story: story),
           SizedBox(height: 16.0.s),
           StoryControlButton(
+            borderRadius: 16.0.s,
+            iconPadding: 8.0.s,
             icon: Assets.svg.iconBlockShare.icon(
               color: context.theme.appColors.onPrimaryAccent,
+              size: 20.0.s,
             ),
-            onPressed: () {
-              StoryContactsShareRoute().push<void>(context);
+            onPressed: () async {
+              ref.read(storyPauseControllerProvider.notifier).paused = true;
+
+              await StoryContactsShareRoute().push<void>(context);
+
+              if (context.mounted) {
```

Comment:
Do we need this check here? There's no `context` interaction.

Reply:
removed